examples/shortguid.rs

```diff
@@ -0,0 +1,81 @@
+use base64::Engine;
+use clap::{Arg, ArgAction, ArgMatches, Command};
+use shortguid::ShortGuid;
+
+fn parse_arguments() -> ArgMatches {
+    let input_id_arg = Arg::new("input_id").help("User input ID").required(true);
+
+    let short_id_arg = Arg::new("short")
+        .short('s')
+        .long("short")
+        .action(ArgAction::SetTrue);
+
+    let long_id_arg = Arg::new("long")
+        .short('l')
+        .long("long")
+        .action(ArgAction::SetTrue)
+        .conflicts_with("short");
+
+    let convert_command = Command::new("convert")
+        .about("Convert the provided id to it's short or default UUID representation")
+        .arg(&input_id_arg)
+        .arg(short_id_arg)
+        .arg(long_id_arg);
+
+    let random_command = Command::new("random")
```

Comment:
Maybe we should call this `new` instead of `random`; the description can stay the same.

Reply:
Sounds reasonable.